SHIR/build.ps1

```diff
@@ -46,6 +46,7 @@ try {
     Install-Jre
     Install-NetFramework
     Install-SHIR
+    SetupEnv
```

Comment:
This just stops the SHIR after the image build, so it'll be started by the container creation setup.ps1 instead.